app/Providers/AppServiceProvider.php

```diff
@@ -13,7 +14,8 @@ class AppServiceProvider extends ServiceProvider
      */
     public function boot()
     {
-        //
+        Schema::defaultStringLength(191);
```

Comment:
This is fine for now, this is due to the multi-byte utf support that's needed. But we need to find a mechanism to go around the default string length problem and move this concert someplace that makes sense.